src/main/resources/sql/insert-products-into-table.sql

```diff
@@ -0,0 +1,30 @@
+INSERT INTO store_project_db.products (id,title,available, price) values (1,'tangerine', 16 ,39.69);
```

Comment:
Assume the product entity include id generated strategy that is not aware about these inserts. On a first application start up you will not be able to add products because the hibernate will try to insert a product record with id 1 but the database already includes a product with such id.

Reply:
remove id from mock insert

Reply:
done ✅